Added 12 new Q&A pairs to the Support knowledgebase section (#1081)
wandb/docs: content/support/how_do_i_login_to_my_server_instance.md

---

> title: How do I login to my server instance?

Reviewer: ```suggestion
title: How can I log in to W&B Server?
```

toc_hide: true
type: docs
tags:
  - user management
---

> To login to a server instance, you should point to its URL by either:

Reviewer: ```suggestion
Set the login URL by either of these methods:

- Set the [environment variable]({{< relref "/guides/track/environment-variables.md" >}}) `WANDB_BASE_URL` to the Server URL.
- Set the `--host` flag of [`wandb login`]({{< relref "/ref/cli/wandb-login.md" >}}) to the Server URL.

```
